test/extended/apiserver/kubeconfigs.go

```diff
@@ -29,6 +30,13 @@ var _ = g.Describe("[Conformance][sig-api-machinery][Feature:APIServer] local ku
 	} {
 		kubeconfig := kc
 		g.It(fmt.Sprintf("%q should be present on all masters and work", kubeconfig), func() {
+			// external controlplane topology doesn't have master nodes
+			controlPlaneTopology, err := exutil.GetControlPlaneTopology(oc)
+			o.Expect(err).NotTo(o.HaveOccurred())
+			if *controlPlaneTopology != configv1.ExternalTopologyMode {
+				g.Skip("ExtternalControlPlaneToplogy doesn't have master node kubeconfigs")
```

Comment:
```suggestion
				g.Skip("ExternalControlPlaneTopology doesn't have master node kubeconfigs")
```

Reply:
Fixed

Reply:
(i'm not trying to be a pain, getting this stuff consistent is useful/important when people start grepping the codebase trying to find references to certain strings)

Reply:
Updated